website/pages/en/arbitrum/l2-transfer-tools-faq.mdx

```diff
@@ -84,6 +94,92 @@ Once transferred, your Ethereum mainnet version of this subgraph will be depreca
 
 Gas fees on Arbitrum One are paid using bridged ETH (i.e. ETH that has been bridged to Arbitrum One). However, the gas fees are significantly lower when compared to Ethereum mainnet.
 
+## Delegation
+
+## How do I transfer my delegation?
```

Comment:
Sorry @MichaelMacaulay, but I think you were correct to change the questions to `###`. This is incorrect to me (even though the linter doesn't catch it), because "Delegation" is technically a heading for the questions below. They are not semantically at the same level.

Reply:
So to be clear, this is what I'm proposing: https://github.com/graphprotocol/docs/pull/512#discussion_r1326287821

Reply:
You're right - it is better that way. Got that in there! Thank you!